packages/loot-core/src/server/accounts/sync.ts

```diff
@@ -61,6 +86,39 @@ async function updateAccountBalance(id, balance) {
   ]);
 }
 
+async function getAccountSyncStartDate(id) {
+  // there is no way to get the current date in sql.js, the workaround is to
+  // calculate the date in js and pass it into the query
+  const currentDate = new Date().toISOString().slice(0, 19).replace('T', ' ');
```

Comment:
Can we use monthUtils here? See months.ts file

Reply:
Found while I was refactoring to aql that this bit isn't neccesary at all, removed now.